flang/test/Lower/Intrinsics/atan_real16.f90

```diff
@@ -6,15 +6,15 @@
 function test_real16(x)
   real(16) :: x, test_real16
   test_real16 = atan(x)
-end function real16
+end function test_real16
 
 ! CHECK-LABEL: @_QPtest_real16
 ! CHECK: fir.call @_FortranAAtanF128({{.*}}){{.*}}: (f128) -> f128
 
 function test_real16_2(y, x)
   real(16) :: y, x, test_real16
   test_real16 = atan(y, x)
-end function real16_2
+end function test_real16_2
 
 ! CHECK-LABEL: @_QPtest_real16
```

Comment:
Should this line contain `@_QPtest_real16_2`?

Reply:
Ugh! Sorry. I should have looked more closely. Fixed that and several other names.